framework/src/main/java/org/tron/core/Wallet.java

```diff
@@ -350,9 +350,14 @@ public TransactionCapsule createTransactionCapsule(com.google.protobuf.Message m
       CreateSmartContract contract = ContractCapsule
           .getSmartContractFromTransaction(trx.getInstance());
       long percent = contract.getNewContract().getConsumeUserResourcePercent();
+      byte[] data = contract.getNewContract().getBytecode().toByteArray();
+
       if (percent < 0 || percent > 100) {
         throw new ContractValidateException("percent must be >= 0 and <= 100");
       }
+      if (ArrayUtils.isEmpty(data)) {
+        throw new ContractValidateException("bytecode must have");
```

Comment:
`s/bytecode must have/bytecode must not be empty/`

Reply:
change this one